github/ci/prow-deploy/molecule/default/cleanup.yml

```diff
@@ -14,3 +14,4 @@
       vars:
         main_actions:
           - cleanup
+      ignore_errors: yes
```

Comment:
Why do we need that here and at other places?

Reply:
Yes, these are added to cleanup and collect actions, we need them to allow all the actions in these tasks to complete, we can't collect all the logs when the cluster fails to spin up otherwise. As I understand it they won't mask any potential problem being tasks related to cleaning up and log collection after the tests, wdyt?

Reply:
OK, thanks for the explanation.